src/components/corrections-and-notices/_macro.njk

```diff
@@ -0,0 +1,79 @@
+{% macro onsCorrectionsAndNotices(params) %}
+    {% set corrections_content = params.corrections %}
+    {% set notices_content = params.notices %}
+
+    {% if corrections_content or notices_content %}
+        <details class="ons-corrections-notices">
+            <summary class="ons-corrections-notices__banner ons-u-pt-s ons-u-pb-s">
+                <div class="ons-container ons-corrections-notices__banner-contents">
+                    <svg class="ons-corrections-notices__info-icon ons-u-mr-xs" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32">
+                        <g id="Artwork_32x32" data-name="Artwork 32x32">
+                            <path
+                                d="M16,2A14,14,0,1,0,30,16,14,14,0,0,0,16,2Zm0,5.75A1.75,1.75,0,1,1,14.25,9.5,1.75,1.75,0,0,1,16,7.75Zm2.5,15.5a.5.5,0,0,1-.5.5H14a.5.5,0,0,1-.5-.5v-1a.5.5,0,0,1,.5-.5h.5v-7H14a.5.5,0,0,1-.5-.5v-1a.5.5,0,0,1,.5-.5h3a.5.5,0,0,1,.5.5v8.5H18a.5.5,0,0,1,.5.5Z"
+                                style="fill:currentColor"
+                            />
+                        </g>
+                    </svg>
+                    <div class="ons-corrections-notices__banner-body">
+                        {% if corrections_content and notices_content %}
+                            <h3 class="ons-corrections-notices__banner-title ons-u-mb-2xs">Corrections and notices</h3>
+                        {% elif corrections_content %}
+                            <h3 class="ons-corrections-notices__banner-title ons-u-mb-2xs">Corrections</h3>
+                        {% else %}
+                            <h3 class="ons-corrections-notices__banner-title ons-u-mb-2xs">Notices</h3>
+                        {% endif %}
+                        <div class="ons-corrections-notices__banner-detail-reveal" aria-hidden="true">
+                            <span class="ons-corrections-notices__banner-detail-text">Show detail</span>
+                            <span class="ons-corrections-notices__banner-detail-icon ons-corrections-notices__banner-detail-icon--show">
```

Comment:
Do we need a separate `--hide` and `--show` for the icon as well, or can we just apply the relevant styles using the [open] attribute?